src/Hl7.Fhir.ElementModel.Tests/SourceNodeTests.cs

```diff
@@ -0,0 +1,180 @@
+﻿/* 
+ * Copyright (c) 2015, Firely (info@fire.ly) and contributors
```

Comment:
2015? Should we not use the year when the file has been created? So 2019?

Reply:
The code is older, but certainly not from 2015 ;-)